### PR TITLE
Release istio v1.9.8

### PR DESCRIPTION
**Pull request by brendarearden**
OOB Release for Istio 1.9.8

The version of Kiali did not get bumped, so there is not a new chart / asset for kiali.

### CODE REVIEW

**Review by cbron — APPROVED** [review]
lgtm, will merge after others approve

**Comment by brendarearden** [comment]
FYI @cbron: per Arvind's suggestion, I did remove the timestamp changes in the index.yaml.

File: index.yaml
Comment (aiyengar2) [inline index.yaml:1309]: kiali annotation looks good; chart exists https://github.com/rancher/charts/tree/release-v2.5/charts/rancher-kiali-server/rancher-kiali-server/1.32.100.

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Pull request merged**
